apps/web/components/Badge.tsx

```diff
@@ -1,48 +1,41 @@
 import React from "react";
-import type { HTMLAttributes } from "react";
-import cn from "classnames";
 
-export type Type = "signaling" | "funding" | "streaming";
-type Status =
-  | "active"
-  | "paused"
-  | "cancelled"
-  | "executed"
-  | "inactive"
-  | string;
+const proposalTypes = ["signaling", "funding", "streaming"];
```

Comment:
the order should be the same as the enum ProposalType inside CVStrategy.sol contract. i think that is "funding", "signaling" and then "streaming".

Reply:
i think that problem is across multiples branches, will approve it and fix in another after all merges.